apps/astarte_appengine_api/test/astarte_appengine_api/device/device_test.exs

```diff
@@ -1004,6 +1021,114 @@ defmodule Astarte.AppEngine.API.DeviceTest do
                par
              ) == {:error, :interface_not_in_introspection}
     end
+
+    test "is successful and data on the interface can be retrieved" do
+      test_realm = "autotestrealm"
+      device_id = "fmloLzG5T5u0aOUfIkL8KA"
+      test_interface = "org.ServerOwnedIndividual"
+      value = 10
+      path = "/1/samplingPeriod"
+      par = %{}
+
+      now = DateTime.utc_now()
```

Comment:
You probably should split this in `now_1` and `now_2` (or probably `request_time_1` and `request_time_2`